configuration/frontend/base-config.json

```diff
@@ -31,5 +31,17 @@
       "generalPatientNoteUuid": "3cd9d956-26fe-102b-80cb-0017a47871b2",
       "midUpperArmCircumferenceUuid": "4326b04b-3158-417a-bb8d-ad022295b0f4"
     }
+  },
+  "@openmrs/esm-patient-medications-app": {
+    "durationUnitsConcept": "3cd706b8-26fe-102b-80cb-0017a47871b2",
+    "daysDurationUnit": {
+      "uuid": "3cd706b8-26fe-102b-80cb-0017a47871b2"
+    },
+    "careSettingUuid": "6f0c9a92-6f24-11e3-af88-005056821db0",
+    "quantityUnitsUuid": "162395AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA",
+    "clinicianEncounterRole": "c458d78e-8374-4767-ad58-9f8fe276e01c",
```

Comment:
Could use a constant for this if desired:  ${encounterRole.orderingProvider.uuid}